pyarlo/camera.py

```diff
@@ -34,6 +34,11 @@ def device_id(self):
         return self._attrs.get('deviceId')
 
     @property
+    def serial_number(self):
+        """Return serial_number."""
+        return self.device_id
```

Comment:
i dunno how useful this is, but it was mentioned in the README, so i thought i'd make it real. 😄 
if this is something we want to keep around, it might make more sense sense to replace the property `serialNumber` checks (e.g. `if cam['serialNumber'] == self.device_id`) with `if cam['serialNumber'] == self.serial_number`

Reply:
Yes, that is a good point.